Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -2811,7 +2811,9 @@ void AXObjectCache::handleAttributeChange(Element* element, const QualifiedName&
     } else if (attrName == colspanAttr) {
         postNotification(element, AXNotification::ColumnSpanChanged);
         recomputeParentTableProperties(element, TableProperty::CellSlots);
-    } else if (attrName == popovertargetAttr)
+    } else if (attrName == commandforAttr)
+        postNotification(element, AXNotification::CommandForChanged);
```

Comment:
Don't we also need to listen for `commandAttr` changes? e.g. if it's removed, or added, the given element's potential to support expanded changes, right?

Reply:
Ah that's a good point, I'll update to account for that and add to the test to check that behaviour.